src/init/Cargo.lock

```diff
@@ -1111,6 +1346,9 @@ name = "spin"
 version = "0.9.8"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "6980e8d7511241f8acf4aebddbb1ff938df5eebe98691418c4468d0b72a96a67"
+dependencies = [
+ "lock_api",
+]
```

Comment:
It's interesting that there is a new dependency for "spin" but we didn't update this package at all.